### PR TITLE
Update copyright text

### PR DESCRIPTION
**Pull request by ScorpioCPH**
@gaocegege @Jimexist @DjangoPeng @ddysher PTAL.

### CODE REVIEW

File: version/version.go [hunk 1]
```diff
@@ -1,9 +1,12 @@
 /*
-Copyright 2018 Caicloud Authors
+Copyright 2017 The Kubernetes Authors.
```
Comment (gaocegege) [inline version/version.go]: I am confued why Kubernetes is the author.

Comment (ddysher) [reply version/version.go]: Copy paste?

Comment (gaocegege) [reply version/version.go]: Alright

Comment (ScorpioCPH) [reply version/version.go]: Update to "Copyright 2017 Caicloud Inc. All Rights Reserved.".

**Pull request merged**